packages/message/src/index.vue

```diff
@@ -112,9 +99,11 @@ export default defineComponent({
     off(document, 'keydown', this.keydown)
   },
   methods: {
+    handleAfterLeave() {
+      this.destroyElement()
+    },
     destroyElement() {
       this.visible = false
```

Comment:
Since `this.visible` is set to `false` in `destroyElement` which would be called in `close` method. Setting `this.visible` here is redundant. Since this `destroyElement` and `handleAfterLeave` only calls `this.onClose`. You only need to pass `onClose` to `after-leave`. That'd be all.

Reply:
OK